api_docs/include/rest-endpoints.md

```diff
@@ -57,19 +57,19 @@
 
 #### Users
 
-* [Get all users](/api/get-users)
-* [Get own user](/api/get-own-user)
 * [Get a user](/api/get-user)
 * [Get a user by email](/api/get-user-by-email)
-* [Update a user](/api/update-user)
-* [Update your status](/api/update-status)
+* [Get all users](/api/get-users)
+* [Get own user](/api/get-own-user)
```

Comment:
nit: swap these two ("get own user" feels logically adjacent to the two other ways of getting a single user)